### PR TITLE
Editorial review on user case doc

### LINKED ISSUE #455: Fix editorial errors in use case doc
### Wflow version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `master` [here](https://github.com/Deltares/Wflow.jl)

### Kind of issue

Docs are unclear

### Location of the documentation

https://github.com/Deltares/Wflow.jl/blob/master/docs/src/intro/use_cases.md

### Documentation problem

While reading the document, I found several typos, editorial errors and some sentences that can be improved.

### Suggested fix for documentation

Check the details in the pull request.

### PR DESCRIPTION
**Pull request by petershintech**
## Issue addressed
Fixes #455 455

## Explanation
Proposed fixes and revisions.

## Checklist
- [ ] Updated tests or added new tests
- [ ] Branch is up to date with `master`
- [ ] Tests & pre-commit hooks pass
- [X] Updated documentation if needed
- [ ] Updated changelog.md if needed

## Additional Notes (optional)
Feel free to select proposed changes when merging.

### CODE REVIEW

File: docs/src/intro/use_cases.md [hunk 1]
```diff
@@ -20,26 +20,26 @@ streamflow predictions.
...
-model results were already quite satisfying, especially given the complex nature of these
-basins, including several small and big reservoirs. We could also demonstrate the potential
-for further calibration by changing the KsatHorFrac parameter.
+The results of this proof of concept are very promising. Technically, we were able to
+quickly set up the Wflow\_sbm models, couple them to the flood inundation models (SFINCS), and
+run the models operationally under the Delft-FEWS platform. Model validation was conducted
+for two basins by comparing the results of Wflow\_sbm against observations and the
+results of calibrated URBS models. This validation demonstrated that the uncalibrated Wflow\_sbm
+model results were already quite satisfactory, especially given the complex nature of these
+basins, which include several small and large reservoirs. We could also showed the potential
```
Comment (verseve) [inline docs/src/intro/use_cases.md:41]:
```suggestion
basins, which include several small and large reservoirs. We could also show the potential
```

File: docs/src/intro/use_cases.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+Reliable hydrological models for the Rhine and the Meuse river basins are necessary for
 short-term forecasting of river flows and long-term predictions for strategic water
-management planning. In collaboration with Rijkswaterstaat, Deltares is setting up a new
+management planning. In collaboration with Rijkswaterstaat, Deltares is developing a new
 line of models for the Rhine and the Meuse basins. The models will be used for forecasting
 and to estimate the impact of climate change on water resources and extreme streamflow. In
-the model development, we seek to improve hydrological predictions by including relevant
-processes in the model schematization. The modularity of the wflow framework is ideal for
+the model development, we aim to improve hydrological predictions by including relevant
+processes in the model schematization. The modularity of the Wflow framework is ideal for
```
Comment (verseve) [inline docs/src/intro/use_cases.md:11]: We decided to use lowercase for `wflow` unless at the start of a sentence or when we mention the Wflow.jl framework (also the case for the GMD paper about the wflow_sbm model).

```suggestion
processes in the model schematization. The modularity of the wflow framework is ideal for
```

File: docs/src/intro/use_cases.md [hunk 3]
```diff
@@ -20,26 +20,26 @@ streamflow predictions.
 
 In Australia, there was a need for high-resolution, fast and accurate rainfall-runoff models
 to provide boundary conditions for a fast and detailed flood inundation model (SFINCS). The
-domain of the flood model covers the complete North and East Coast of Australia. Although
+domain of the flood model covers the entire North and East Coast of Australia. Although
 many gauging stations are available to provide real-time information, many rivers are not
-covered. For these locations, wflow\_sbm models are used to provide this real-time
+covered. For these locations, Wflow\_sbm models are used to provide this real-time
```
Comment (verseve) [inline docs/src/intro/use_cases.md:25]:
```suggestion
covered. For these locations, wflow\_sbm models are used to provide this real-time
```

File: docs/src/intro/use_cases.md [hunk 4]
```diff
@@ -20,26 +20,26 @@ streamflow predictions.
...
 information. Additionally, these models are used to provide projections for potential future
-scenarios. Using the HydroMT library, all wflow\_sbm models were automatically built. The
-high level of flexibility in spatial and temporal resolution, and the physics-based nature
-of the concept makes wflow\_sbm a very suitable model for ungauged basins. Furthermore, the
-model is sufficiently detailed and computationally efficient for a coupling with the fast
+scenarios. Using the HydroMT library, all Wflow\_sbm models were automatically built. The
```
Comment (verseve) [inline docs/src/intro/use_cases.md:27]:
```suggestion
scenarios. Using the HydroMT library, all wflow\_sbm models were automatically built. The
```

File: docs/src/intro/use_cases.md [hunk 5]
```diff
@@ -20,26 +20,26 @@ streamflow predictions.
...
-The results of this Proof of Concept are very promising. Technically, we were able to very
-quickly setup the wflow\_sbm models, couple them to the flood inundation models (SFINCS) and
-run the models operationally under the Delft-FEWS platform. Model validation was carried out
-for two basins by comparing the results of wflow\_sbm against the observations and the
-results of calibrated URBS models. This validation showed that the uncalibrated wflow\_sbm
-model results were already quite satisfying, especially given the complex nature of these
-basins, including several small and big reservoirs. We could also demonstrate the potential
-for further calibration by changing the KsatHorFrac parameter.
+The results of this proof of concept are very promising. Technically, we were able to
+quickly set up the Wflow\_sbm models, couple them to the flood inundation models (SFINCS), and
```
Comment (verseve) [inline docs/src/intro/use_cases.md:36]:
```suggestion
quickly set up the wflow\_sbm models, couple them to the flood inundation models (SFINCS), and
```

File: docs/src/intro/use_cases.md [hunk 6]
```diff
@@ -51,30 +51,30 @@ Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.
...
 movement of plastic waste is in many cases triggered by runoff from (heavy) rainfall and
-transported by the flow of water towards small streams and rivers. Therefore there is strong
+transported by water flow towards small streams and rivers. Therefore there is strong
 relation to rainfall-runoff processes, which can be modeled using high-resolution
 rainfall-runoff models.
 
-In this study we applied the wflow\_sbm model in combination with a fate-and-transport and
-water quality model (DelWaq) to simulate the movement of plastics through 5 large river
-basins and on 3 island and coastal zones (Krabi, Phuket and Ko Samui, see screenshot of the
+In this study we applied the Wflow\_sbm model in combination with a fate-and-transport and
```
Comment (verseve) [inline docs/src/intro/use_cases.md:63]:
```suggestion
In this study we applied the wflow\_sbm model in combination with a fate-and-transport and
```

File: docs/src/intro/use_cases.md [hunk 7]
```diff
@@ -51,30 +51,30 @@ Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.
...
-to show hotspots of plastic polution, how much plastic waste would end up in the Gulf of
-Thailand and what would be priority areas for reducing plastic waste reaching the sea.
+to identify hotspots of plastic pollution, estimate how much plastic waste would end up in the Gulf of
+Thailand and recommend priority areas for reducing plastic waste reaching the sea.
 
 ![fig_case_mfa](../images/case_mfa_1.png)
 
-The wflow\_sbm models for the 5 large basins were calibrated. The presence of large dams and
-reservoirs complicated the calibration, but with the input for the operation of the dams,
+The Wflow\_sbm models for the five large basins were calibrated. The presence of large dams and
```
Comment (verseve) [inline docs/src/intro/use_cases.md:72]:
```suggestion
The wflow\_sbm models for the five large basins were calibrated. The presence of large dams and
```

File: docs/src/intro/use_cases.md [hunk 8]
```diff
@@ -51,30 +51,30 @@ Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.
...
+reservoirs complicated calibration, but with the input for the dam operation,
 the model performance for these basins could be largely improved. The figure below shows the
 calibrated model results for the Chao Phraya, just upstream of Bangkok. The input from the
-hydrological wflow\_sbm model was used as input for the fate and transport model to assess
-how much plastic would be transported to the ocean.
+hydrological Wflow\_sbm model was used as input for the fate and transport model to assess
```
Comment (verseve) [inline docs/src/intro/use_cases.md:76]:
```suggestion
hydrological wflow\_sbm model was used as input for the fate and transport model to assess
```

**Review by verseve — CHANGES_REQUESTED** [review]
Thanks for this editorial review on the use cases!
Some small comments, mainly about using lowercase for wflow.

**Comment by petershintech** [comment]
@verseve I assumed 'Wflow' is mentioned with the capital 'W' but I might be wrong. All your suggestions were accepted. As I read other docs, I might propose similar suggestions if I can. Thanks for sharing the source code of this model.

**Comment by verseve** [comment]
> @verseve I assumed 'Wflow' is mentioned with the capital 'W' but I might be wrong. All your suggestions were accepted. As I read other docs, I might propose similar suggestions if I can. Thanks for sharing the source code of this model.

Yes, the Julia package starts indeed with a capital `W` (naming convention of Julia packages). This package is a continuation of a Python package (https://github.com/openstreams/wflow) and then (quite some time ago) it was decided to use lowercase letters. We decided to continue with this convention (for now).

**Review by verseve — APPROVED** [review]

**Pull request merged**